### PR TITLE
Updated SourceKitten to 0.3.2

### PR DESCRIPTION
**Pull request by jpsim**

### CODE REVIEW

File: spec/integration_spec.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@
       behaves_like cli_spec 'document_alamofire',
                             '-m Alamofire -a Alamofire ' \
                             '-u https://nshipster.com/alamofire ' \
-                            '-x -project,Alamofire.xcodeproj ' \
+                            '-x -project,Alamofire.xcodeproj,-dry-run ' \
```
Comment (segiddins) [inline spec/integration_spec.rb]: wait, I thought using `-dry-run` didn't work?

**Comment by jpsim** [comment]
Clickable link to integration test fixtures: https://github.com/realm/jazzy-integration-specs/commit/dc1c3da2fbe1cc4ddc3f6b36fb2ecfe6b44e8657

File: spec/integration_spec.rb (hunk 1 above)
Comment (jpsim) [inline spec/integration_spec.rb]: using `-dry-run` by default won't work for cases where there are non-apple dependencies, but that's not the case for either Alamofire or MiscJazzyFeatures.

**Comment by segiddins** [comment]
@jpsim this is now green

**Pull request merged**